test/functional/basic/performance-e2e-specs.js

```diff
@@ -23,7 +23,7 @@ describe('XCUITestDriver - performance', function () {
     });
 
     it('should return recorded trace file on stop', async function () {
-      if (process.env.CI) {
+      if (process.env.CI || process.env.CLOUD) {
```

Comment:
Isn't the cloud stuff always also in CI? That is, is it ever the case the `process.env.CI` is falsy and `process.env.CLOUD` is truthy?

Reply:
Only when testing locally